opam

```diff
@@ -32,8 +32,14 @@ build-test: [
   [make]
   ["./test.sh"]
 ]
-install: [make "install"]
-remove:  ["ocamlfind" "remove" "cstruct"]
+install: [
+  [make "install"]
+  [make "js-install"]
```

Comment:
should this be depopted on `js_of_ocaml`, or is it not worth the bother (just install it unconditionally)